packages/js-moi-manifest/src.ts/manifest.ts

```diff
@@ -279,4 +280,182 @@ export class ManifestCoder {
+                        WireType.WIRE_WORD,
+                        ...element.data,
+                    ]);
+                    schema = Schema.PISA_TYPEDEF_SCHEMA;
+                    break;
+                }
+
+                case "state": {
+                    buff = new Uint8Array([
+                        WireType.WIRE_PACK,
+                        ...element.data,
+                    ]);
+                    schema = Schema.PISA_STATE_SCHEMA;
+                    break;
+                }
+
+                case "routine": {
+                    buff = new Uint8Array([
+                        WireType.WIRE_PACK,
+                        ...element.data,
+                    ]);
+                    schema = Schema.PISA_ROUTINE_SCHEMA;
+                    break;
+                }
+
+                case "method": {
+                    buff = new Uint8Array([
+                        WireType.WIRE_PACK,
+                        ...element.data,
+                    ]);
+                    schema = Schema.PISA_METHOD_SCHEMA;
+                    break;
+                }
+
+                case "class": {
+                    buff = new Uint8Array([
+                        WireType.WIRE_PACK,
+                        ...element.data,
+                    ]);
+                    schema = Schema.PISA_CLASS_SCHEMA;
+                    break;
+                }
+
+                case "event": {
+                    buff = new Uint8Array([
+                        WireType.WIRE_PACK,
+                        ...element.data,
+                    ]);
+                    schema = Schema.PISA_EVENT_SCHEMA;
+                    break;
+                }
+
+                default: 
+                    ErrorUtils.throwError(`Unsupported kind: ${element.kind}`, ErrorCode.UNSUPPORTED_OPERATION);
+            }
+
+            if (schema == null || buff == null) {
+                ErrorUtils.throwError("Invalid schema or buffer", ErrorCode.UNEXPECTED_ARGUMENT);
+            }
+
+            const polo = new Depolorizer(buff);
+            element.data = polo.depolorize(schema);
+
+            if (["routine", "method"].includes(element.kind)) {
+                for (const [key, value] of Object.entries(
+                    element.data.executes
+                )) {
+                    // If value of `Instruction[key]` is empty, remove it
+                    if (value === "") {
+                        delete element.data.executes[key];
+                    }
+
+                    // If value of `Instruction` is empty, remove it
+                    if (Array.isArray(value) && value.length === 0) {
+                        delete element.data.executes[key];
+                    }
+
+                    // Required to convert Uint8Array to integer array
+                    if (value instanceof Uint8Array) {
+                        element.data.executes[key] = Array.from(value);
+                    }
+                }
```

Comment:
Why are we converting it?

Reply:
<img width="465" alt="image" src="https://github.com/user-attachments/assets/9e97170e-fe9f-4cf2-b07c-95d341395994">

As the type of bin is bytes during depolorize. Converting it to json will give the following result.

<img width="412" alt="image" src="https://github.com/user-attachments/assets/d7cb06ba-e83a-4406-948e-9091128f1062">

As in the type of this is number array. Thus it required to convert it

<img width="498" alt="image" src="https://github.com/user-attachments/assets/7a625155-4cae-417f-89f7-2591b65dae3e">